### ISSUE TITLE
Cannot add recipes for Custom Extract flakes with MineTweaker

### ISSUE THREAD

**Issue by Falgar** [ReikaKalseki/Reika_Mods_Issues#3150]
I updated from v30c to v31b and realized that I can no longer melt the Metallurgy ore flakes that I added via the Custom Extract config in Tinkers' Construct Smelteries. I added these recipes using MineTweaker and they had previously worked. I used the same 1.5x rate and appropriate temperatures (as much as possible for fantasy metals). Given that these items already have their entire multiplication bonuses left to the user, preventing further customization seems unintended.

I had also added smeltery recipes for Eximite and Meutoite flakes (as RotaryCraft includes those ores but doesn't provide smeltery recipes), for Nether Ores (these should just be equivalent to their overworld counterparts eg Nether Iron Flakes vs Iron Ore Flakes), and for the Silver Flakes that are the byproduct of Gold Ore extraction (these are somehow different than those produced by Silver Ore extraction and were missing a recipe). However, these probably shouldn't be added by MineTweaker, but rather fixed in RotaryCraft.

**Comment by ReikaKalseki** [comment]
I am not understanding what you are trying to say.

**Comment by Falgar** [comment]
I added custom extraction for all of the metallurgy ores, for example Rubracium. The flakes generated from this (eg "Rubracium flakes") are given the standard furnace recipes to convert to ingots (or their appropriate items), but not Smeltery recipes. Previously, I had added Smeltery recipes for the flakes, allowing me to melt the flakes in Tinkers Construct Smelteries, as native RotaryCraft flakes can be. However, in some intervening update, that no longer works. The recipes are commented out in the script files citing that they're not allowed to be modified.

Also, RotaryCraft is missing some Smeltery recipes for its own natively added flakes, namely Meuotite, Eximite, and the Silver flakes from Gold Ore extraction. These recipes were also commented out when I tried to fix them.
